keyman_test.go

```diff
@@ -39,15 +39,15 @@ func TestRoundTrip(t *testing.T) {
 	assert.NoError(t, err, "Unable to load PK")
 	assert.Equal(t, pk.PEMEncoded(), pk2.PEMEncoded(), "Loaded PK didn't match saved PK")
 
-	cert, err := pk.TLSCertificateFor(time.Now().Add(TWO_WEEKS), true, nil, "Test Org", "CommonName", "testdomain.com", "127.0.0.1")
+	cert, err := pk.TLSCertificateFor(time.Now().Add(TWO_WEEKS), true, nil, "Test Org", "TestCommonName", "testdomain.com", "127.0.0.1")
 	assert.NoError(t, err, "Unable to generate self-signed certificate")
 
 	numberOfDNSSans := len(cert.X509().DNSNames)
-	if numberOfDNSSans != 1 {
+	if numberOfDNSSans != 2 {
 		t.Errorf("Wrong number of DNS SANs, expected 1 got %d", numberOfDNSSans)
 	} else {
```

Comment:
I think this test broke because of [this change](https://github.com/getlantern/keyman/commit/cfd0ee278507db710453dd85706535f05f411707#diff-bbf7bc3c19fb78f00afb17a3faeb835509631d81c306cf23119d712f5b3be290R199)

Reply:
Sorry about that, not sure how I forgot to run the tests!  Thanks for the fix.

Reply:
Not sure how I didn't catch it during review :)